extensions/objects/templates/numeric-expression-editor.component.ts

```diff
@@ -19,104 +19,135 @@
 // Every editor directive should implement an alwaysEditable option. There
 // may be additional customization options for the editor that should be passed
 // in via initArgs.
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
```

Comment:
There seems to be some issue with the event emission here:
![bug](https://user-images.githubusercontent.com/35144226/116228591-1b896980-a773-11eb-9389-390fe6debe7a.gif)

Reply:
Solved.... this was due to the fact currentValue was being set in the change and focus event listeners after the validation. Changed it.